config/vector.toml

```diff
@@ -12,33 +12,71 @@
 # Chat: https://chat.vector.dev
 # ------------------------------------------------------------------------------
 
-# Change this to use a non-default directory for Vector data storage:
-# data_dir = "/var/lib/vector"
-
-# Random Syslog-formatted logs
-[sources.dummy_logs]
-type = "demo_logs"
-format = "syslog"
-interval = 1
-
-# Parse Syslog logs
-# See the Vector Remap Language reference for more info: https://vrl.dev
-[transforms.parse_logs]
-type = "remap"
-inputs = ["dummy_logs"]
-source = '''
-. = parse_syslog!(string!(.message))
-'''
-
-# Print parsed logs to stdout
-[sinks.print]
-type = "console"
-inputs = ["parse_logs"]
-encoding.codec = "json"
-
-# Vector's GraphQL API (disabled by default)
-# Uncomment to try it out with the `vector top` command or
-# in your browser at http://localhost:8686
-#[api]
-#enabled = true
-#address = "127.0.0.1:8686"
+# The directory used for persisting Vector state.
+data_dir = "/var/lib/vector"
+
+# Configure Vector's GraphQL API.
+[api]
+enabled = true
+address = "[::]:8686"
+playground = false
+
```

Comment:
I am thinking that for ease of "quickstart" having an `http` source may be nice even if it's not totally practical to have a default config for such a generic component.

Without it though, there's not an super simple way to get an event into Vector in an Aggregator role (`curl` in an event).

Reply:
👍 I'd be down with an `http` source configured to accept ndjson.